freqtrade/exchange/__init__.py

```diff
@@ -269,9 +269,28 @@ def get_ticker(pair: str, refresh: Optional[bool] = True) -> dict:
 
 
 @retrier
-def get_ticker_history(pair: str, tick_interval: str) -> List[Dict]:
+def get_ticker_history(pair: str, tick_interval: str, since: Optional[int] = None) -> List[Dict]:
     try:
-        return _API.fetch_ohlcv(pair, timeframe=tick_interval)
+        # download data until it reaches today now time
+        #
+        # it looks as if some exchanges return cached data
+        # and update it with some delay so 10 mins interval is added
+        data = []
+        while not since or since < arrow.utcnow().shift(minutes=-10).timestamp * 1000:
```

Comment:
I just want to prevent doing the extra call. So 10 minutes is just a delay to adjust a logic for cashes of exchanges API. From my testing it looks like 10 minutes is enough for exchanges to update the data. 

1h+ tickers needs some other checks. Will add them.

Reply:
The documentation says that `fetch_ohlcv` will return the data that starts after since time. So there should not be any records duplications.

Reply:
works for me - just asked as it was unclear to me :+1: